docs/tutorials/multi_contrast.ipynb

```diff
@@ -200,60 +202,120 @@
     "roughness = 3"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "e50a2ffd",
+   "metadata": {},
+   "source": [
+    "The `'d13DSPC-D2O'` surfactant layer "
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "021272e0-7abb-4703-a4cd-92daed10ae50",
+   "id": "c4aa6c00",
    "metadata": {},
    "outputs": [],
    "source": [
-    "d13d2o = SurfactantLayer.from_pars(\n",
-    "    tail_layer_molecular_formula=dspc['h-tail'], \n",
-    "    tail_layer_thickness=tail_thickness,\n",
-    "    tail_layer_solvent=d2o, \n",
-    "    tail_layer_solvent_fraction=tail_solvent_fraction, \n",
-    "    tail_layer_area_per_molecule=area_per_molecule,\n",
-    "    tail_layer_roughness=roughness,\n",
-    "    head_layer_molecular_formula=dspc['d-head'],\n",
-    "    head_layer_thickness=head_thickness, \n",
-    "    head_layer_solvent=d2o,\n",
-    "    head_layer_solvent_fraction=head_solvent_fraction, \n",
-    "    head_layer_area_per_molecule=area_per_molecule,\n",
-    "    head_layer_roughness=roughness\n",
+    "tail_layer_d13d2o = LayerAreaPerMolecule(\n",
+    "    molecular_formula=dspc['h-tail'],\n",
+    "    thickness=tail_thickness,\n",
+    "    solvent=air,\n",
+    "    solvent_fraction=tail_solvent_fraction,\n",
+    "    area_per_molecule=area_per_molecule,\n",
+    "    roughness=roughness\n",
+    ")\n",
+    "head_layer_d13d2o = LayerAreaPerMolecule(\n",
+    "    molecular_formula=dspc['d-head'],\n",
+    "    thickness=head_thickness,\n",
+    "    solvent=d2o,\n",
+    "    solvent_fraction=head_solvent_fraction,\n",
+    "    area_per_molecule=area_per_molecule,\n",
+    "    roughness=roughness\n",
+    ")\n",
+    "d13d2o = SurfactantLayer(\n",
+    "    tail_layer=tail_layer_d13d2o,\n",
+    "    head_layer=head_layer_d13d2o\n",
     ")\n",
     "d13d2o.constrain_area_per_molecule = True\n",
     "d13d2o.conformal_roughness = True\n",
-    "d13d2o.constrain_solvent_roughness(d2o_layer)\n",
-    "d70d2o = SurfactantLayer.from_pars(\n",
-    "    tail_layer_molecular_formula=dspc['d-tail'],\n",
-    "    tail_layer_thickness=tail_thickness,\n",
-    "    tail_layer_solvent=d2o,\n",
-    "    tail_layer_solvent_fraction=tail_solvent_fraction, \n",
-    "    tail_layer_area_per_molecule=area_per_molecule,\n",
-    "    tail_layer_roughness=roughness,\n",
-    "    head_layer_molecular_formula=dspc['h-head'],\n",
-    "    head_layer_thickness=head_thickness,\n",
-    "    head_layer_solvent=d2o,\n",
-    "    head_layer_solvent_fraction=head_solvent_fraction, \n",
-    "    head_layer_area_per_molecule=area_per_molecule,\n",
-    "    head_layer_roughness=roughness\n",
+    "d13d2o.constrain_solvent_roughness(d2o_layer)"
```

Comment:
those json-like entries mean that you probably didn't clean the notebook? Please consider resetting all cells before merging

Reply:
notebooks have been cleaned